### ISSUE TITLE
Can't compile the code

### ISSUE THREAD

**Issue by legomannetje** [SensorsIot/IOTAppstory.com#9]
At first thank you for making all those good video's! And more thanks for this wonderfull project!

All the headerfile are in place, but the arduino IDE still says its not there.
How to fix it?

Thanks in advance!

**Comment by SensorsIot** [comment]
Can you provide a little more details, please?

**Comment by legomannetje** [comment]
I tried to compile the virgin soil example, i copied before opening the arduino IDE the credentials.h en ESP_helpers.h to the folder. When i tried to compile the file i get that those files don't exist

![afbeelding](https://cloud.githubusercontent.com/assets/11525684/21934876/6af3070a-d9ab-11e6-88cb-89868a6be4bb.png)

The errormessage:

C:\Users\***\Documents\Arduino\IOT\virginSoil\virginSoil.ino:36:25: fatal error: credentials.h: No such file or directory

 #include <credentials.h>

                         ^

compilation terminated.

exit status 1
Fout bij het compileren van board NodeMCU 1.0 (ESP-12E Module)

**Comment by caiosweet** [comment]
Try like this...
in the folder user libraries creates 2 subfolders
"Credentials"
"ESP_helpers"
put .h files within the respective subfolders
Folder libraries --> subfolder ESP_helpers -> ESP_helpers.h
Folder libraries --> subfolder credentials -> credentials.h

**Comment by legomannetje** [comment]
Thanks! it works now! 

But i dont think this is the way it's suppost to be.....

**Comment by caiosweet** [comment]
It depends on the source ...

If you want to include in the same folder you have to change
`#include <credentials.h> to #include "credentials.h"`
`#include <ESP_Helpers.h> to #include "ESP_Helpers.h"`

From what I understood,
for `#include "filename"` search in the same folder as the main program (usually used by the programmer)
for `#include <filename>` search according to the compiler directives (usually used for libraries)

**Comment by legomannetje** [comment]
@SensorsIot maybe you can change this in the examples, or explane it somewhere that the extra files has to be in a library folder! this is something i couldn't find that easly

@caiosweet Thanks for your help!

**Comment by caiosweet** [comment]
Happy to be helpful. 
I lost a bit 'of time to figure out how it worked. in a video, I forget which, you see A.S. insert the ESP_helpers folder in the user library .. maybe, using this method, since such a file is shared with multiple projects? 
I also think that it is useful to add in the project folder, so it is easier to have it under control ...
 unfortunately for me it's hard to follow you, do not speak English, but someone could add a few lines to aid in the description of the project files here on github (README.md)
